tests/callbacks/test_early_stopping.py

```diff
@@ -205,3 +207,82 @@ def validation_epoch_end(self, outputs):
     )
     trainer.fit(model)
     assert trainer.current_epoch >= 5, 'early_stopping failed'
+
+
+@pytest.mark.parametrize('step_freeze', [5, 6, 7])
+@pytest.mark.parametrize('min_steps', list(range(4, 8)))
+@pytest.mark.parametrize('min_epochs', list(range(2, 3)))
+def test_min_steps_override_early_stopping_functionality(tmpdir, step_freeze, min_steps, min_epochs):
+    """Excepted Behaviour:
+    IF `min_steps` was set to a higher value than the `trainer.global_step` when `early_stopping` is being triggered,
+    THEN the trainer should continue until reaching `trainer.global_step` == `min_steps`, and stop.
+    This test validate this expected behaviour
+    """
+
+    warnings.filterwarnings("ignore")
```

Comment:
what warnings do you filter here?

Reply:
I am trying to prevent 
```
Epoch 75:  60%|███████████████Trainer was signaled to stop but required minimum epochs (2) or minimum steps (100) has not been met. Training will continue...0<00:00, 152.18it/s, loss=0.167, v_num=0]
```
to pollute the logs.
But it doesn't seem to work :) Thanks for catching that up :)

Reply:
to filter those, I think the filter must be applied before the other imports

Reply:
This works 👍 
```
from pytorch_lightning import _logger
_logger.disabled = True
_logger.disabled = False
```